desktop-widgets/preferences/preferences_language.cpp

```diff
@@ -22,20 +22,36 @@ PreferencesLanguage::PreferencesLanguage() : AbstractPreferencesWidget(tr("Langu
 	filterModel->sort(0);
 	connect(ui->languageFilter, &QLineEdit::textChanged,
 			filterModel, &QSortFilterProxyModel::setFilterFixedString);
+
+	dateFormatShortMap.insert("MM/dd/yyyy", "M/d/yy");
+	dateFormatShortMap.insert("dd/MM/yyyy", "d/M/yy");
+	dateFormatShortMap.insert("yyyy-MM-dd", "yy-M-d");
```

Comment:
Isn’t day.month.year usually written with a dot? Or at least alternatively? In Germany today is 10.11.2017

Reply:
`10.11.2017` is also what i use in BG. can be added.